imports/plugins/included/ui-search/client/templates/searchModal/searchModal.js

```diff
@@ -169,15 +168,14 @@ Template.searchModal.helpers({
     const brand = __.pluck(result, ["vendor"]);
```

Comment:
I wouldn't want us to import `lodash` and `underscore` in the same project. Check lodash docs and you'd find what you're looking for. 😀